src/modules/m_spanningtree/main.cpp

```diff
@@ -705,9 +705,8 @@ namespace
 			if (!linkstring.empty())
 				buffer << '=' << linkstring;
 
-			const std::string compatlinkstring = Utils->BuildLinkString(PROTO_INSPIRCD_3, mod);
```

Comment:
This is actually a bug. The line below should be checking if `compatlinkstring` is empty not `linkstring`.